internal/xds/bootstrap/tlscreds/bundle_ext_test.go

```diff
@@ -237,6 +238,7 @@ func (s) TestCaReloading(t *testing.T) {
 // is performed and checked for failure, ensuring that gRPC is correctly using
 // the changed-on-disk bundle map.
 func (s) Test_SPIFFE_Reloading(t *testing.T) {
+	testutils.SetEnvConfig(t, &envconfig.XDSSpiffeEnabled, true)
```

Comment:
And same here. If we are going to guard this with the env var, we need to have a test case where the env var is set to `false`.